proxy-support/README.md

```diff
@@ -0,0 +1,273 @@
+- The entire monitoring and alerting stack needs access to `opsgenie`, `grafana`, ...
+- App platform components need access to an OCI artifact system (e.g. `github.com`, `azure`)
+
+As we already use `kyverno` in our `MC`, we can use the [`kyverno-policies-connectivity`] chart to inject the proxy configuration into all pods.
+
+> Please note: 
+> - due multiple valid reasons, `kyverno` doesn't mutate objects in `namespace/kube-system` per default.
+> - static pods are managed by the `kubelet`. For these pods it's not possible to use `kyverno` for mutation. Therefore it's required to inject the proxy configuration during the machine creation phase (could be achieved in `CAPI` based clusters via `kubeadm`)
+
+#### special workload
+
+There are some components, which are part of the default set on top of a cluster. These components can't get configured via `kyverno` as they are either in `namespace/kube-system` (for `cert-manager`) or must exist before we apply applications via the app platform (for `chart-operator`).
+
+##### `cert-manager`
+
+As `cert-manager` is needed in every cluster (`MC` and `WC`) and it's running in `namespace/kube-system`. `cert-manager` interact with external `ACME` services (`lets encrypt` at the moment).
+The proxy configuration must be explicitly defined in the `pod.spec`.
+
+##### `chart-operator`
+
+`chart-operator` is getting deployed in every `WC` by `cluster-apps-operator` and is responsible for applying charts in the workload cluster.
+The proxy configuration must be explicitly defined in the `pod.spec`. 
+
+## `no_proxy`/`NO_PROXY`
+
+Due possible security policies, customers wants components using a proxy server for external systems. But there might be also internal systems where no proxy server should be used. For that it's possible to define a comma separated list of domain names, FQDNs, network addresses and network ranges.
+
+As within a Kubernetes cluster there are also some targets where no communication via the proxy is expected (e.g.: accessing Kubernetes services within the cluster, directly accessing pods via their IP address - primarily used by `prometheus` for metrics scraping).
+
+For that, the values of `no_proxy` are separated into two logical parts:
+
+- endpoints which are installation wide like a customer internal container registry or a customer internal network range.
+- endpoints which are cluster specific like the `podCIDR`/`serviceCIDR` or the cluster internal base domain (e.g. `cluster.local`)
+
+> Note: the interpretation of the `no_proxy` variable is very language/tool specific and not every language supports e.g. network ranges.
+> a very details blog post was written by GitLab about [proxy variables].
+
+## Implementation
+
+### Management Cluster
+
+#### during the initial bootstrap
+
+During the initial bootstrap phase of the `MC`, there isn't the entire App Platform running. For that reason it's required to define the proxy configuration multiple times.
```

Comment:
Can list the components here? I know it is not a static list and we may change it in the future but it may be good to know in this phase.